### ISSUE TITLE
Random World Crashes

### ISSUE THREAD

**Issue by HackFox-zz** [WTFGeeks/Natural-Magic#105]
I've had several worlds just randomly crash and not load again when wandering around. Log Below. Any help would be awesome, thank you in advance.
http://paste.atlauncher.com/view/2d378225

**Comment by WTFGeeks** [comment]
I'm having a hard time figuring this one out, as it points only to structure generation and not to what the structure is or it's mod that it comes from, the line in the error is the generic Minecraft generation line.  have you created a world and had this problem since Doom Like  Dungeons was removed? the only real way of telling what caused the issue upon gen would be to load the world into a NBT editor and look at it in there.

**Comment by HackFox-zz** [comment]
Yes that log is from the most recent update and a fresh world. I'd have no idea what to look for but I think I can upload the nbt data to a Google Doc

**Comment by Bloodfrost** [comment]
I am having the same issues.  However mine is not from a fresh world, looking at the log he posted, its almost identical to mine.  I tried removing a few of the generation mods manually to try it? like roguelike but it didnt fix it.  My friend said on Journymaps he was heading towards a large black structure before it killed the server if that helps at all...

**Comment by WTFGeeks** [comment]
So I'v been tearing my hair out over this one, I can't seem to replicate this crash so tracking it down is very hard, also the crash logs tell me nothing except during structure generation it placed either blockgrass or block staticliquid both vanilla blocks and crashed, this gives me nothing to go off. As I can't replicate it yet this does mean it's not what is generating otherwise replicating would be easy but where it's generating the structure in turn making it very hard to catch as structure generation is mostly random. Once I can get the game to crash I can use NBT explorer to open the save and see what the chunk was trying to load. All I can suggest currently until I replicate this is to open the world in NBT explorer and have a look yourself or it mite show up in mcedit very unlikely though as the structure should not have loaded.

**Comment by HackFox-zz** [comment]
Ive taken a look several times when I've gotten a chance but much like yourself I'm unable to pinpoint what's causing the issue. I started several new worlds to try to get a cleaner looking log but it always seem to be the same

**Comment by HackFox-zz** [comment]
I gave up entirely and started a new world :(
